TESTS/integration/stress/bogdan.txt

```diff
@@ -0,0 +1,15974 @@
+The Project Gutenberg EBook of Dracula, by Bram Stoker
+
+This eBook is for the use of anyone anywhere at no cost and with
+almost no restrictions whatsoever.  You may copy it, give it away or
+re-use it under the terms of the Project Gutenberg License included
+with this eBook or online at www.gutenberg.org/license
```

Comment:
I'll check  this license if it's OK to have it in

Reply:
I will remove the eBooks under license and replace them with Lorem Ipsum generated files.

Reply:
@bulislaw @0xc0170 eBooks replaced now with random generated Lorem Ipsum data files.